builtin/settingtypes.txt

```diff
@@ -891,8 +891,19 @@ default_privs (Default privileges) string interact, shout
 #    Privileges that players with basic_privs can grant
 basic_privs (Basic privileges) string interact, shout
 
-#    If enabled, disable cheat prevention in multiplayer.
-disable_anticheat (Disable anticheat) bool false
+#    Enable digging cheats prevention in multiplayer.
+anticheat_digging (Prevent fast/instant digging cheats) bool true
+
+#    Enable interaction anticheat in multiplayer.
+anticheat_interacting (Enable anticheat for player interaction) bool true
+
+#    Enable fast movement cheats prevention in multiplayer.
+anticheat_movement (Prevent movement cheats) bool true
```

Comment:
I think a flags field is better suited than separate settings

Reply:
Oh, it's a good idea. I need to learn how to implement it properly though :smile: Gonna look at csm_restriction_flags as the sample...